packages/component/src/Styles/StyleSet/SpinnerAnimation.js

```diff
@@ -12,7 +12,7 @@ export default function createSpinnerAnimationStyle({
   // since defaultStyleOptions has a predefined value for spinnerAnimationPadding by default, the console.warn will appear if both it and spinnerAnimationPaddingRight are set
   if (spinnerAnimationPadding && spinnerAnimationPaddingRight) {
     console.warn(
-      'botframework-webchat: "spinnerAnimationPaddingRight" is being deprecated and will be removed on or after 2021-02-01. Please use "spinnerAnimationPadding" instead.'
+      'botframework-webchat: "spinnerAnimationPaddingRight" is deprecated and will be removed on or after 2021-02-01. Please use "spinnerAnimationPadding" instead.'
```

Comment:
Do we need to add removal dates for the deprecations in this file?

Reply:
The deprecation date is already in the file and is 2021-02-01.